test/common.js

```diff
@@ -51,19 +51,17 @@ function rmdirSync(p, originalEr) {
   }
 }
 
-function refreshTmpDir() {
-  if (!process.send) { // Not a child process
-    try {
-      rimrafSync(exports.tmpDir);
-    } catch (e) {
-    }
+exports.refreshTmpDir = function() {
+  try {
+    rimrafSync(exports.tmpDir);
+  } catch (e) {
+  }
```

Comment:
Hmm. I'd say the comment makes it a little bit different, but i'll leave it up to you if the linter is happy either way.